template.tpl

```diff
@@ -247,28 +247,39 @@ ___TEMPLATE_PARAMETERS___
     ]
   },
   {
-    "type": "TEXT",
-    "name": "accessToken",
-    "displayName": "API Access Token",
-    "simpleValueType": true,
-    "help": "Set to your Facebook API Access Token. See \u003ca href\u003d\"https://developers.facebook.com/docs/marketing-api/server-side-api/get-started#access-token\" target\u003d\"_blank\"\u003ehere\u003c/a\u003e for more information.",
-    "valueValidators": [
+    "type": "SIMPLE_TABLE",
```

Comment:
This solution will have breaking changes for all existing setups. 
Can we enter Ppixel IDs and Access Tokens separated by a comma to existing fields?
I understand managing relations between Pixel IDs and AccessTokens can be terrible, but it will allow us to upgrade smoothly.

Reply:
Done.